### ISSUE TITLE
Should sudo idevicerestore --erase --latest work from restore mode?

### ISSUE THREAD

**Issue by ParetoOptimalDev** [libimobiledevice/libimobiledevice#1440]
I have a 2016 ipad pro.

I ran:

```
sudo idevicerestore --erase --latest
```
It successfully erased the current iOS install but then I got an error.

Now I have an ipad that boots into restore mode with the support.apple.com/ipad/restore link at the top and itunes icon.

I tried running the command above again both in this restore mode and in what I assume was DFU mode (or the ipad was just off, no way to verify I know of). 

It just fails saying it couldn't connect to the device.

Is this expected? What are my options to recover this device?

**Comment by nikias** [comment]
Yes it is supposed to work... can you run with -d so it prints some debug output?
